resources/versioning-policy.md

```diff
@@ -28,6 +28,8 @@ Examples:
 - Change a final value type.
 Ex: `/stats` now return floats instead of integers. This can impact strongly typed languages.
 
+⚠️ This guide only applied to the Meilisearch binary. Additional tools like SDKs and Docker images are not related. However, we will ensure the changelogs are clear enough to inform users of the changes and their impacts.
```

Comment:
```suggestion
⚠️ This guide only applies to the Meilisearch binary. Additional tools like SDKs and Docker images are out of the scope of this guide. However, we will ensure the changelogs are clear enough to inform users of the changes and their impacts.
```

Reply:
Docker images are only not integral to this guide because of the experimental/prototypes tag names, right? Otherwise, I think they could!

Reply:
I'm not sure to understand your question @brunoocasali, what do you mean?

Reply:
When you say this guide does not affect the Docker image versioning, is it just because of some exceptional tag names? 

Because, for me, the rules in this policy can also be applied to Docker, WDYT?